### ISSUE TITLE
k-NN box

### ISSUE THREAD

**Issue by darabos** [lynxkite/lynxkite-2000#297]
Scikit-learn has a ton of great algorithms under a uniform interface. We should make sure it's easy to use them from LynxKite.

As a test case of this, let's implement k-nearest neighbors.
https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html

The way I think this could look:
- Create feature vectors. For categorical features we can offer a one-hot encoding box, like in the old LynxKite. The vector bundling box already exists.
- Create a train/test split.
- Fit the kNN model on the training set.
- Make predictions on the test set.
- Show a plot about the accuracy.

Peter Zubcsek is also interested in kNN.
